.github/workflows/Sweave2rmd.yaml

```diff
@@ -16,34 +17,45 @@ jobs:
       R_KEEP_PKG_SOURCE: yes
       REPO_NAME: ${{ github.event.repository.name }}
     steps:
+
       - uses: actions/checkout@v3
-      - name: create repo
-        run: |
-          echo $REPO_NAME
-          git clone "https://git.bioconductor.org/packages/$REPO_NAME"
-          rm -rf $REPO_NAME/.git*
-          mv $REPO_NAME/* .
-          rm -rf $REPO_NAME
-      - uses: r-lib/actions/setup-r@v2
-        with:
-          use-public-rspm: true
-      - uses: r-lib/actions/setup-r-dependencies@v2
-        with:
-          extra-packages: LiNk-NY/Rnw2Rmd
+
+      - name: install Rnw2Rmd 
+        run: Rscript -e "BiocManager::install('LiNk-NY/Rnw2Rmd')"
+
       - name: convert Rnw
+        id: conversion
         run: |
-          R -e 'setwd("vignettes")
-                 for (fname in list.files(path = ".", pattern = "\\.rnw$", ignore.case = TRUE)){
-                  try(Rnw2Rmd::Rnw2Rmd(from = fname, to = sub("rnw", "Rmd", x = fname, ignore.case = TRUE), validate = FALSE))
-                 }'
+             R -e 'setwd("vignettes")
+                   msgs <- ""
+                   for (fname in list.files(path = ".", pattern = "\\.rnw$", ignore.case = TRUE)) {
+                     msg <- tryCatch({
+                       Rnw2Rmd::Rnw2Rmd(from = fname,
+                                        to = sub("rnw", "Rmd", x = fname, ignore.case = TRUE),
+                                        validate = FALSE)
+                       paste(fname, "converted.")
+                     }, error = function(e) {
+                       paste(fname, "must be converted manually.")
+                     })
+                     msgs <- c(msgs, msg)
+                   }
+                   cat(msgs, file="results.txt")'
+             results=$(cat vignettes/results.txt)
+             rm vignettes/results.txt
+             echo "results=$results" >> $GITHUB_OUTPUT
+
       - name: git add
         run: |
           git config --global --add safe.directory /__w/$REPO_NAME/$REPO_NAME
           git config user.name helpful-bot
           git config user.email github-actions@github.com
-          git add *
-          git rm -r .github
-          git commit -am "PR autogenerated by Sweave2rmd.yaml"
+          git rm  *.Rnw
```

Comment:
I think this should probably be `git rm */vignettes/*.Rnw` otherwise it can remove .Rnw files in other paths.